language/resource-api/README.md

```diff
@@ -184,7 +185,7 @@ A missing `:should` entry indicates that a resource should be removed from the s
 
 The `set` method should always return `nil`. Any progress signaling should be done through the logging utilities described below. If the `set` method throws an exception, all resources that should change in this call and haven't already been marked with a definite state, will be marked as failed. The runtime will only call the `set` method if there are changes to be made, especially in the case of resources marked with `noop => true` (either locally or through a global flag). The runtime will not pass them to `set`. See `supports_noop` below for changing this behaviour if required.
 
-Both methods take a `context` parameter which provides utilties from the runtime environment, and is decribed in more detail there.
+Both methods take a `context` parameter which provides utilities from the runtime environment, and is described in more detail there.
```

Comment:
Where is this described? In the runtime environment?

Reply:
yeah. changed it (and the three other occurrences) to "described in more detail in its own section below"